RiotSwiftUI/Modules/Room/VoiceBroadcastPlayback/MatrixSDK/VoiceBroadcastPlaybackViewModel.swift

```diff
@@ -281,7 +296,9 @@ class VoiceBroadcastPlaybackViewModel: VoiceBroadcastPlaybackViewModelType, Voic
             audioPlayer?.pause()
             displayLink.isPaused = true
         } else {
-            // Flush the current audio player playlist
+            // Flush the chunks queue and the current audio player playlist
+            voiceBroadcastChunkQueue = []
+            reloadVoiceBroadcastChunkQueue = true
```

Comment:
@yostyle you should turn on this flag only when `isProcessingVoiceBroadcastChunk` is true
Don't you think so?